src/mo_frontend/typing.ml

```diff
@@ -394,12 +394,16 @@ let timer_type =
     [Func (Local, Returns, [], [Prim Nat64], [])],
     [Async (Fut, Var (default_scope_var, 0), unit)]))
 
+let low_memory_type =
```

Comment:
Just a heads-up, these type definitions will live in `type.ml` after #4846. I'll take care of this in the migration.